spec/index.html

```diff
@@ -1875,10 +1875,10 @@ <h3>SPARQL</h3>
         [[[SPARQL11-QUERY]]].</p>
     </section>
     <section id="rel-n3-trig">
-      <h3>TriG</h3>
+      <h3>TriG vs N3 Named Graphs</h3>
```

Comment:
This addition does not make much sense to me, as each section pertains to the similarity of N3 with another language (or aspect thereof). In this vein, adding "Named Graphs" as a subsection could work.